testsuite/mass-and-rinertia_per_particle.py

```diff
@@ -14,256 +14,176 @@
         # or both translational and rotational ones.
         # Otherwise these types of motion will interfere.
         # ..Let's test both cases depending on the particle index.
-        gamma_tran[0, 0] = np.array(0.5 + np.random.random())
-        gamma_tran[0, 1] = gamma_tran[0, 0]
-        gamma_tran[0, 2] = gamma_tran[0, 0]
-        gamma_rot[0, :] = np.array((0.5 + random(3)) * 2.0 / 3.0)
+        self.gamma_tran_p[0, 0] = 0.5 + np.random.random()
+        self.gamma_tran_p[0, 1] = self.gamma_tran_p[0, 0]
+        self.gamma_tran_p[0, 2] = self.gamma_tran_p[0, 0]
+        self.gamma_rot_p[0, :] = (0.5 + random(3)) * 2.0 / 3.0
+        self.gamma_tran_p[1, 0] = 0.5 + np.random.random()
+        self.gamma_tran_p[1, 1] = self.gamma_tran_p[1, 0]
+        self.gamma_tran_p[1, 2] = self.gamma_tran_p[1, 0]
+        self.gamma_rot_p[1, 0] = (0.5 + np.random.random()) * 2.0 / 3.0
+        self.gamma_rot_p[1, 1] = self.gamma_rot_p[1, 0]
+        self.gamma_rot_p[1, 2] = self.gamma_rot_p[1, 0]
         
-        gamma_tran[1, 0] = np.array(0.5 + np.random.random())
-        gamma_tran[1, 1] = gamma_tran[1, 0]
-        gamma_tran[1, 2] = gamma_tran[1, 0]
-        gamma_rot[1, 0] = np.array((0.5 + np.random.random()) * 2.0 / 3.0)
-        gamma_rot[1, 1] = gamma_rot[1, 0]
-        gamma_rot[1, 2] = gamma_rot[1, 0]
-
-        if test_case != 4:
-            self.es.thermostat.set_langevin(
-                kT=0.0,
-                gamma=[
-                    gamma_global[0],
-                    gamma_global[1],
-                    gamma_global[2]])
-        else:
-            self.es.thermostat.set_langevin(
-                kT=0.0,
-                gamma=[
-                    gamma_global[0],
-                    gamma_global[1],
-                    gamma_global[2]],
-                gamma_rotation=[
-                    gamma_global_rot[0],
-                    gamma_global_rot[1],
-                    gamma_global_rot[2]])
-
-        self.es.cell_system.skin = 5.0
-        mass = 12.74
-        J = [10.0, 10.0, 10.0]
-
-        for i in range(len(self.es.part)):
-            self.es.part[i].remove()
-
+        ## Particles
+        self.mass = 12.74
+        self.J = 10.0,10.0,10.0
         for i in range(2):
-            self.es.part.add(rotation=(1,1,1), pos=np.array([0.0, 0.0, 0.0]), id=i)
-            self.es.part[i].v = np.array([1.0, 1.0, 1.0])
+            self.es.part.add(rotation=(1,1,1), pos=(0.0,0.0,0.0), id=i)
+            self.es.part[i].v = 1.0,1.0,1.0
             if "ROTATION" in espressomd.features():
-                self.es.part[i].omega_body = np.array([1.0, 1.0, 1.0])
-            self.es.part[i].mass = mass
-            self.es.part[i].rinertia = np.array(J)
-
-        print("\n")
+                self.es.part[i].omega_body = 1.0,1.0,1.0
+            self.es.part[i].mass = self.mass
+            self.es.part[i].rinertia = self.J
 
+    def fluctuation_dissipation_param_setup(self,n):
```

Comment:
could you please use the numpy-style in-code documentation that is used for python functions in espresso and describe what the argument to the function means and what type it has